lib/service/service.go

```diff
@@ -3849,6 +3849,7 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 	}
 
 	tlscfg := serverTLSConfig.Clone()
+	setupTLSConfigClientCAsForCluster(tlscfg, accessPoint, clusterName)
```

Comment:
probably not a big deal but `setupTLSConfigClientCAsForCluster` sets `tlsClone.ClientAuth = tls.VerifyClientCertIfGiven` which conflict with next line.